packages/flutter/test/cupertino/bottom_tab_bar_test.dart

```diff
@@ -11,9 +11,16 @@ import '../widgets/semantics_tester.dart';
 
 Future<void> pumpWidgetWithBoilerplate(WidgetTester tester, Widget widget) async {
   await tester.pumpWidget(
-    Directionality(
-      textDirection: TextDirection.ltr,
-      child: widget,
+    Localizations(
+      locale: const Locale('en', 'US'),
+      delegates: <LocalizationsDelegate<dynamic>>[
+        DefaultWidgetsLocalizations.delegate,
+        DefaultCupertinoLocalizations.delegate,
+      ],
+      child: Directionality(
+        textDirection: TextDirection.ltr,
+        child: widget,
+      ),
     ),
   );
 }
```

Comment:
can you add one test for the semantics content of the tabs?

Reply:
There is already a test for that which was checking the value of the hard-coded string, now it is checking the localized result. :) 
https://github.com/flutter/flutter/blob/0549ab239009e6f76f95881364e935cd345a4da7/packages/flutter/test/cupertino/bottom_tab_bar_test.dart#L360